.github/dependabot.yml

```diff
@@ -2,10 +2,27 @@ version: 2
 updates:
 - package-ecosystem: github-actions
   directory: "/"
+  labels:
+    - "Build"
   schedule:
     interval: daily
 - package-ecosystem: nuget
   directory: "/src"
+  labels:
+    - "Build"
+  groups:
+    analyzers:
+      patterns:
+      - "*Analyzers"
+    serilog:
+        patterns:
```

Comment:
How picky is yml again?

Isn't this line intended too far?

Reply:
yes, fixed